versions.props

```diff
@@ -4,7 +4,7 @@ com.carrotsearch:hppc=0.9.1
 com.cybozu.labs:langdetect=1.1-20120112
 com.fasterxml.jackson:jackson-bom=2.13.3
 com.fasterxml.woodstox:woodstox-core=6.2.8
-com.github.ben-manes.caffeine:caffeine=3.0.5
```

Comment:
Also need to update versions.lock and the license files. `./gradlew helpDeps` has the commands you need.